### DISCUSSION TITLE
I'd like to know waht gets synced with spotify and which way?

### DISCUSSION THREAD

**Discussion by ovizii** [KRTirtho/spotube#808] [General]
The main page states:

> Access all your Spotify Music & Playlists
> You can use all your Spotify tracks & playlists here. Everything will be in Sync & some action taken from Spotube will also reflect on your Spotify Account.

So I am wondering what exactly is "some action". 

The initial reason for the question was that after following more and more artists via spotify and via spotube, it looks like only on spotify can I see the newly auto-generated artist playlists so I was wondering whether it just takes time for spotube to sync those to be available via spotube too?

edit:
I am also wondering if "recommended stations" will end up synced down to spotube?

**Comment by KRTirtho** [comment]
Except playback info & podcasts, everything starting from Playlists, favorite tracks, artists, albums will be in Sync
"Some actions" include creating/updating (add/remove track) playlists, adding track to favorite
